typescript/src/purchasesConfiguration.ts

```diff
@@ -52,5 +52,5 @@ export interface PurchasesConfiguration {
    * Verification strictness levels for [EntitlementInfo].
    * See https://rev.cat/trusted-entitlements for more info.
    */
-  entitlementVerificationInfo?: ENTITLEMENT_VERIFICATION_MODE;
+  entitlementVerificationMode?: ENTITLEMENT_VERIFICATION_MODE;
```

Comment:
This is technically a breaking change, but I would argue the old name didn't make sense, so it's a fix.